install_sct

```diff
@@ -528,7 +520,7 @@ if [ ${SCT_DEV_MODE} ]; then
   pip install -e ${SCT_SOURCE}
   e_status=$?
 else
-  pip install ${SCT_SOURCE}
+  pip install -e ${SCT_DIR}
```

Comment:
do we still need to distinguish dev vs. non-dev mode? i.e., maybe we don't need the `-z` flag anymore?

Reply:
If nobody uses it, it could be removed. The only justification for its existence is that with it, we can perform out-of-place installs and edit files that are not in the installation folder (but I have never done that).

Reply:
I think we can remove it.

Reply:
OK done.